users.md

```diff
@@ -218,7 +218,7 @@ Ritual is a mobile pick up app, connecting restaurants with customers to offer a
 
 ### [Snap](https://www.snap.com/en-US/)
 
-Snap, the developper of Snapchat messaging app, has migrated from Buck to Bazel in 2020 ([source](https://twitter.com/wew/status/1326957862816509953)).
+Snap, the developper of Snapchat messaging app, has migrated from Buck to Bazel in 2020 ([source](https://twitter.com/wew/status/1326957862816509953)). Check out their [engineering blog](https://eng.snap.com/blog/) for more about their process.
```

Comment:
Editorial feedback. Ending statements with the link is easier for users. Rephrase to "For more details about their process, see their [engineering blog](https://eng.snap.com/blog/).

Reply:
Done